nautobot/extras/models/jobs.py

```diff
@@ -248,6 +248,12 @@ def class_path(self):
 
     @property
     def latest_result(self):
+        """
+        Return the most recent JobResult object associated with this Job.
+
+        Note that, as a performance optimization for this function's repeated use in
+        JobListview, the returned object only includes its `status` field.
```

Comment:
Is this accurate? I don't see `.only("status")` at line 252/258 below?

Reply:
Nope. Fixing. Bad merge choice.